pkg/vpcmodel/detailedConn.go

```diff
@@ -105,15 +105,19 @@ func (e *detailedConn) tcpRspNonTCPComponent() *connection.Set {
 	return e.tcpRspEnable.Union(e.nonTCP)
 }
 
-// todo: remove once transformation is completed, rename enhanceString to string
+// todo: will it still be needed once transformation is completed?
 func (e *detailedConn) string() string {
 	if !e.tcpRspDisable.IsEmpty() {
 		return e.allConn.String() + asterisk
 	}
 	return e.allConn.String()
 }
 
-func (e *detailedConn) enhanceString(bidirectional bool) string {
+// in the structs a single line represents connection of each <src, dst>
+// in the reports we print potentially two lines for each <src, dst> connection:
+// one for the "main" tcp responsive + non tcp component and the other for the tcp non-responsive component
+// this separation is done here: the former is returned for bidirectional and the latter for false
+func (e *detailedConn) connStrPerConnectionType(bidirectional bool) string {
```

Comment:
consider changing input arg name to `nonResponsiveTCP` instead of `bidirectional` (and then flip the boolean check)

Reply:
True should be for the more common case... shouldn't it?

Reply:
not sure, but I find it confusing to be called "bidirectional" when it only refers to the TCP responsive part , and other protocols without any context of bidirectional.

Reply:
so how about `responsiveTCPAndNonTCP`?

Reply:
maybe `nonTCPAndResponsiveTCP` (so it is clear that "responsive" only refers to TCP)

Reply:
done, also in another similar case